edge-proxy/rpm/edge-proxy.spec

```diff
@@ -11,7 +11,7 @@ License:        Apache-2.0
 URL:            %{gourl}
 Source0:        %{gosource}
 
-BuildRequires:  golang >= 1.14, golang < 1.15
+BuildRequires:  golang < 1.15
```

Comment:
Alternatively will this work? https://serverfault.com/a/723503

Reply:
Hrmm.... seems like a bug that Redhat doesn't allow you to specify a version range.  Weird.